lib/chai-subset.js

```diff
@@ -1,51 +1,80 @@
-module.exports = function(chai, utils) {
-	var Assertion = chai.Assertion;
-	var assertionPrototype = Assertion.prototype;
-
-	Assertion.addChainableMethod('containSubset', function (expected) {
-		var actual = utils.flag(this, 'object');
-		var showDiff = chai.config.showDiff;
-
-		assertionPrototype.assert.call(this,
-			compare(expected, actual),
-			'expected #{act} to contain subset #{exp}',
-			'expected #{act} to not contain subset #{exp}',
-			expected,
-			actual,
-			showDiff
-		);
-	});
-};
-
-function compare(expected, actual) {
-	if (typeof(actual) !== typeof(expected)) {
-		return false;
-	}
-	if (typeof(expected) !== 'object' || expected === null) {
-		return expected === actual;
-	}
-	if (!!expected && !actual) {
-		return false;
+(function () {
+  "use strict";
```

Comment:
Do you have exact reasons why to use strict mode? If no, i would like to not have this in code